Lesson32/selcan/my-app/src/shared-components/typography/BodyS/index.jsx

```diff
@@ -0,0 +1,8 @@
+import './styles.css';
+
+export const BodyS = ({title, bold}) => {
+
+  return (
+    <h2 className={bold ? 'bold' : ''}>{title}</h2>
```

Comment:
Since this component is to display normal text, not heading, use `<p>` element instead.